apps/src/templates/sectionAssessments/SectionAssessments.jsx

```diff
@@ -87,12 +88,63 @@ class SectionAssessments extends Component {
     asyncLoadAssessments(sectionId, scriptId);
   }
 
-  onChangeScript = scriptId => {
-    const {setScriptId, asyncLoadAssessments, sectionId} = this.props;
-    asyncLoadAssessments(sectionId, scriptId);
-    setScriptId(scriptId);
+  onSelectScript = newScriptId => {
+    const {setScriptId, asyncLoadAssessments, scriptId, sectionId} = this.props;
+    asyncLoadAssessments(sectionId, newScriptId);
+    setScriptId(newScriptId);
+
+    this.logEvent('select_script', {
+      old_script_id: scriptId,
+      new_script_id: newScriptId
+    });
+  };
+
+  onSelectAssessment = newAssessmentId => {
+    const {setAssessmentId, assessmentId, scriptId} = this.props;
+    setAssessmentId(newAssessmentId);
+
+    this.logEvent('select_assessment', {
+      script_id: scriptId,
+      old_level_group_id: assessmentId,
+      new_level_group_id: newAssessmentId
+    });
   };
 
+  onSelectStudent = studentId => {
+    const {setStudentId, assessmentId, scriptId} = this.props;
+    setStudentId(studentId);
+
+    this.logEvent('select_student', {
+      student_id: studentId,
+      script_id: scriptId,
+      level_group_id: assessmentId
+    });
+  };
+
+  onClickDownload(dataType) {
+    const {assessmentId, scriptId} = this.props;
+    this.logEvent('download_results', {
```

Comment:
I think putting all of the different download type in the same event might make it more difficult to query for, say, just the events related to downloading assessment data (right?).  Do we want to put the dataType as part of the event name (e.g. "download_surveys", "download_feedback", etc.) instead?

Reply:
this is what Amanda asked for but i'll double check

Reply:
Amanda agrees, will update!